### PR TITLE
[terra-functional-testing] Add browsers and grid url options to the test runner cli

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary

This PR adds the browsers and grid url options to the test runner cli.

```
npm run terra -- wdio --browsers chrome firefox --gridUrl grid.test.com
```

### Additional Details

Some of the previous chrome logging information has been deprecated and/or had API changes. This PR does not carry forward the chrome performance logging.

Many of these changes are carried forward from here:

https://github.com/cerner/terra-toolkit-boneyard/blob/main/config/wdio/selenium.config.js

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

**Review by yuderekyu — APPROVED** [review]
small, but can we update our `about` documentation to remove the `--` from the example terra-cli script? i think that confuses most people who aren't developing in this mono repo

https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

File: packages/terra-functional-testing/src/config/capabilities.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * This file contains the capability configurations for each supported browser.
+ */
+
+module.exports = {
+  chrome: {
```
Comment (benbcai) [inline packages/terra-functional-testing/src/config/capabilities.js:6]: Do we no longer need to set [loggingPrefs](https://github.com/cerner/terra-toolkit-boneyard/blob/ea4a5fc159f8e5f71911077ef613259694940e47/config/wdio/selenium.config.js#L6-L8) and [perfLoggingPrefs](https://github.com/cerner/terra-toolkit-boneyard/blob/ea4a5fc159f8e5f71911077ef613259694940e47/config/wdio/selenium.config.js#L15-L17)? I don't know who looks at the performance logs or what they are used for.

Comment (StephenEsser) [reply packages/terra-functional-testing/src/config/capabilities.js:6]: I left a short description in the PR summary about the logging. Some of the APIs have changed/been removed and I wasn't able to find any concrete documentation on how the logging was actually intended to be used. I haven't seen any examples of people using the logging, so I think this is something we can passively look into adding back in later (after some investigation) if necessary.

Here's a few related issues I stumbled across while looking around:

https://github.com/webdriverio/webdriverio/issues/590 
https://github.com/webdriverio/webdriverio/issues/476
http://v4.webdriver.io/guide/testrunner/browserobject.html
https://chromedriver.chromium.org/logging/performance-log

Comment (benbcai) [reply packages/terra-functional-testing/src/config/capabilities.js:6]: Oops, I didn't pay attention to your details on the PR summary.

**Review by benbcai — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**